CMakePresets.json

```diff
@@ -43,6 +43,13 @@
         "CMAKE_CXX_CLANG_TIDY": "clang-tidy;--header-filter=${sourceDir}/*"
       }
     },
+    {
+      "name": "include-what-you-use",
+      "hidden": true,
+      "cacheVariables": {
+        "CMAKE_CXX_INCLUDE_WHAT_YOU_USE": "include-what-you-use;-Xiwyu;--mapping_file=${sourceDir}/iwyu.imp;-Xiwyu;--no_fwd_decls"
```

Comment:
Why is `--no_fwd_decls` needed? Aren't they preferable to including the whole header file?

Reply:
We talked about it. It's fine.